content/guides/print-connection/step1-who-does-what.md

```diff
@@ -0,0 +1,40 @@
+# Who does what?
+
+## Macro overview
+
+![Macro overview of the print connection](../../img/guides/print-connection-macro.svg)
+
+Akeneo PIM provides a single place to collect, manage, and enrich product information, create a product catalog, and distribute it to your print solution.
+
+In the context of a "PIM/print" architecture, the **PIM becomes the Reference of marketing product information**.
+
+
+## Akeneo PIM and DAM
```

Comment:
I don't see the link between this section and this step. Why do we need to talk about the PIM/DAM relation in the documentation for PIM/print connection?